src/tools/auth0/handlers/scimHandler.ts

```diff
@@ -125,7 +117,19 @@ export default class ScimHandler {
 			return await callback();
 		} catch (error) {
 			const errorData = error?.response?.data;
-			if (errorData?.statusCode === 404) throw "SCIM_NOT_FOUND";
+      // Skip the connection if it returns 404. This can happen if `SCIM` is not enabled on a `SCIM` connection.
+			if (errorData?.statusCode === 404) {
+        const warningMessage = `SCIM configuration is not enabled on connection \"${ this.currentConnectionId }\".`;
+        log.warn(warningMessage);
+        return { data: null };
+      };
+
+      // Skip the connection if it returns 403. Looks like "scim_config" permissions are not enabled on Management API. 
+			if (errorData?.statusCode === 403) {
+        const warningMessage = `SCIM configuration is not enabled on connection \"${ this.currentConnectionId }\".`;
+        log.warn(warningMessage);
+        return { data: null };
+      }
```

Comment:
After this change will we still be able to manage connections via IaC even if we don't use SCIM?

Reply:
Hello @bweisberger ,

Yes. That's correct !

Reply:
Ok, I ask because after looking [here](https://github.com/auth0/auth0-deploy-cli/blob/master/src/tools/auth0/handlers/connections.ts#L102) I got the impression that all connections updates use the scimHandler, which seems to be skipping connections updates [here](https://github.com/auth0/auth0-deploy-cli/pull/925/files#diff-38cb0d98b14b46a15ddcd2a6398c0b97fabd410e28af0e5b10579402e3359c68R66) if SCIM updates fail. I'm sure I'm missing plenty, but just wanted to check

Reply:
If a SCIM operation encounters an error other than `404` or `403`, it could potentially cause the connection to fail. While a `429` error doesn't necessarily mean immediate failure, our goal is to avoid triggering 429 errors altogether if possible. Because, if SCIM causes `429`, it'll fail connection as well.